jsonschema-hyperschema.xml

```diff
@@ -425,7 +425,7 @@
                 <section title="Submitting data for processing">
                     <t>
                         The <xref target="submissionSchema">"submissionSchema"</xref> and
-                        <xref target="submissionEncType">"submissionEncType"</xref> keywords
+                        <xref target="submissionMediaType">"submissionMediaType"</xref> keywords
                         describe the domain of the processing function implemented by the target
                         resource. Otherwise, as noted above, the submission schema and encoding are
```

Comment:
The line above still talks about "encoding". Maybe change to media type. (This is the only place in the document where "encoding" is used, AFAICT.)

Reply:
Good catch.  I'd mostly done search-and-replace with a bit of skimming, it did not occur to me to search for "encoding" for some reason.